packages/core/__test__/registries/CustomStyleRegistry.test.js

```diff
@@ -39,14 +39,14 @@ describe('TestCustomStyleRegistry', function () {
     expect(fetchedCssString).to.equal('');
   });
 
-  it('Test define twice same name', async function () {
+  it('Test define twice same name', function () {
     CustomStyleRegistry.define(testName, mockCssString);
 
     try {
```

Comment:
we should use `expect` with callback similar to `packages/core/__test__/decorators/CustomElement.test.js` to test that there is no error thrown in non localhost environment as well.